ui/app/components/selectable-card.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+/**
+ * @module SelectableCard
+ * SelectableCard components are card-like components that display a title, total, subtotal, and anything after they yield.
+ * They are designed to be used in containers that act as flexbox or css grid containers.
+ *
+ * @example
+ * ```js
+ * <SelectableCard @cardTitle="Tokens" @total={{totalHttpRequests}} @subText="Total" @gridContainer={{gridContainer}}/>
+ * ```
+ * @param {string} cardTitle - cardTitle displays the card title
+ * @param {number} total - the Total number displays like a title, it's the largest text in the component
+ * @param {string} subText - subText describes the total
+ * @param {string} gridContainer - Optional parameter used to display CSS grid item class.
+ */
+
+export default Component.extend({
+  tagName: '', // do not wrap component with div
+  cardTitleComputed: computed('total', function() {
+    let cardTitle = this.cardTitle || '';
+    let total = this.total || '';
+
+    if (cardTitle === 'Tokens') {
+      return total !== 1 ? 'Tokens' : 'Token';
+    } else if (cardTitle === 'Entities') {
+      return total !== 1 ? 'Entities' : 'Entity';
+    }
+
+    return cardTitle;
+  }),
+  cardTitleTesting: computed('cardTitle', function() {
```

Comment:
OK, this might be one we can pair through.  Basically, I am doing this because I can't test the two words "Http Requests". using the following in the `selectable-card-container.test`:  ```assert.dom(`[data-test-selectable-card-title=Http Requests]`).exists();```

Reply:
I could do a work around like the following, but it would only be for "Http Requests".
```
let httpRequestsTitle = this.element.querySelector('[data-test-selectable-card-title]').innerText;
    // we have to test the Http Requests title this way because it's two words, unlike Entities and Tokens.
    assert.equal(httpRequestsTitle, 'Http Requests');
```